### PR TITLE
First draft of Emojify command

### PR DESCRIPTION
**Pull request by Pspritechologist**
No idea if I'm doing this right :)

(Also I couldn't get it to build locally because of some config.json file and I wouldn't know how to test it anyway but it ran fine online :))

### CODE REVIEW

File: src/utils/emojifyFunc.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
+const rI = 'regional_indicator_'; // Prefix for regional indicators, just to save on space.
+
+const fallBackEmoji = ':joker:'; // Wildcard, baby :)
+
+const zWJ = '\u200D'; // This is required to stop emoji from making little emoji babies. Otherwise :regional_indicator_c::regional_indicator_a: would become 🇨🇦
```
Comment (Pspritechologist) [inline src/utils/emojifyFunc.ts:5]: I figure this should paste into Discord fine, but I also have it as a character here >> ‌ <<

**Comment by Stanford47** [comment]
I'll make a small update that will help with contributing :) (sorry)

that should give you some time to flesh this out more :3

**Review by Stanford47 — CHANGES_REQUESTED** [review]
The code looks fine and should work, but, there are 3 things that I would change. (Just me being really nit-picky)

### emojify.ts
- The comma on line 13 should be put on a new line (for formatting reasons)

### emojifyFunc.ts
- The const variable `zWJ` on line 5 isn't necessary, since the zero width joiner is used once in the code (line 148).
- You should use the actual emoji characters when possible, Discord accepts that. example: `"circled_m"` can just be `Ⓜ️` and still work.

That's all :3

**Comment by Pspritechologist** [comment]
> ### emojifyFunc.ts
> * The const variable `zWJ` on line 5 isn't necessary, since the zero width joiner is used once in the code (line 148)

Using random strings in the middle of functions without documentation is bad practice and I will not be taking part in it, thank you very much.

> * You should use the actual emoji characters when possible, Discord accepts that. example: `"circled_m"` can just be `Ⓜ️` and still work.

I personally liked having the raw text, but if you insist; It is Emojified~

**Review by Stanford47 — APPROVED** [review]
Thanks :) (you could totally just add a comment about zWJ in the function but it's not that important) approved!

**Pull request merged**